chef/cookbooks/ceph/recipes/radosgw.rb

```diff
@@ -61,6 +64,20 @@
   include_recipe "ceph::radosgw_keystone"
 end
 
+admin_tenant_id = keystone_settings["admin_tenant_id"]
+
+# add caps to user admin to access the bucketst 
+if node.roles.include?("ceph-radosgw")
+  cmd = "radosgw-admin caps add --uid=#{admin_tenant_id} --caps='buckets=*'"
+  set_bucket_caps = Chef::ShellOut.new(cmd)
+  Chef::Log.debug("Executing command: #{cmd}")
+  set_bucket_caps.run_command
+  cmd = "radosgw-admin caps add --uid=#{admin_tenant_id} --caps='users=*'"
+  set_users_caps = Chef::ShellOut.new(cmd)
+  Chef::Log.debug("Executing command: #{cmd}")
+  set_users_caps.run_command
```

Comment:
Also I am pretty sure that ```read``` caps (instead of ```*```) should be enough here. (But I didn't verify that)